source/optionalize.d.ts

```diff
@@ -0,0 +1,30 @@
+/**
+The sister of `Require` type. Allows you to define one or more keys for a type that should be optional.
```

Comment:
> Allows you to define one or more keys for a type that should be optional.

This is not entirely correct as it creates a new type. It doesn't modify the existing type.